web/themes/contrib/civictheme/webpack/webpack.common.js

```diff
@@ -9,13 +9,15 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 module.exports = {
   entry: (function (pattern) {
     // Splitting entries into three chunks:
-    // main: all styles used in components and drupal theme -> output: civictheme.css
-    // variables: CSS variables -> output: civictheme.variables.css
-    // editor: nested styles used in editor -> output: civictheme.editor.css
+    // - main: all styles used in components and Drupal theme -> civictheme.css
+    // - variables: CSS variables -> civictheme.variables.css
+    // - editor: nested styles used in editor -> civictheme.editor.css
     const entries = {
       main: [],
       variables: [],
       editor: [],
+      layout: [],
+      admin: [],
```

Comment:
this is where we introduce a new "admin" chunk to be imported as a library for authenticated users. this is used for cases when admin UI elements are shown when CivicTheme is active.